crates/storage/db/src/metrics.rs

```diff
@@ -20,14 +20,31 @@ const LARGE_VALUE_THRESHOLD_BYTES: usize = 4096;
 #[derive(Debug)]
 pub struct DatabaseEnvMetrics {
     /// Caches OperationMetrics handles for each table and operation tuple.
-    operations: HashMap<(Tables, Operation), OperationMetrics, BuildHasherDefault<FxHasher>>,
+    operations: FxHashMap<(Tables, Operation), OperationMetrics>,
+    /// Caches TransactionMetrics handles for counters grouped by only transaction mode.
+    /// Updated both at tx open and close.
+    open_transactions: FxHashMap<TransactionMode, TransactionMetrics>,
```

Comment:
we record both opens and closes of the transactions using these metrics, so I think such naming is better, similar to `operations`

```suggestion
    transactions: FxHashMap<TransactionMode, TransactionMetrics>,
```

Reply:
Fixed.